common/icons/package.json

```diff
@@ -5,10 +5,10 @@
   "sideEffects": false,
   "scripts": {
     "compile": "yarn run-svgr && yarn move-index",
-    "move-index": "mv components/index.tsx ./index.ts",
+    "move-index": "mv components/index.ts ./index.ts",
     "run-svgr": "svgr svgs --out-dir components --no-svgo --typescript --template template.js --index-template index-template.js"
```

Comment:
I ran `run-svgr` and it created a new file (that errors because of the paths): `common/icons/components/index.ts`

Should we adapt this command line/not run it individually? (it's the one we're told to run in the `components/*.tsx` files)

Reply:
oh strange - I'm not getting an error regardless of whether I run `yarn compile` or `yarn run-svgr` – what is the error?

Reply:
It's not an error per say, it just creates that file and it looks like this

<img width="620" alt="Screenshot 2024-07-17 at 14 11 33" src="https://github.com/user-attachments/assets/22395464-2d06-4dd1-945e-8943d4b4bf39">

Look like `move-index` is the one that puts it in the correct directory? So should `run-svgr` never be run by itself?

Reply:
oh right, yeah – I see what you mean. I can update the comment

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/icons/index-template.js#L9